client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -347,6 +347,36 @@ describe('Edit Required Information', () => {
     });
 
     expect(getByText(/Submit/)).toBeDefined();
+  });  
+ 
+  it('should pass multiple valid phone numbers and emails', async () => {
+    const stateWithFields = {
+      ...testState,
+      fields: {
+        countries: [[230, 'United States']],
+        institutions: [[1, 'University of Texas at Austin']],
+        ethnicities: [['Decline', 'Decline to Identify']],
+        genders: [['Other', 'Other']],
+        professionalLevels: [['Other', 'Other']],
+        titles: [['Other User', 'Other User']],
+      }
+    };
+    const storeWithFields = mockStore({ profile: stateWithFields });
+    stateWithFields['data']['demographics']['phone'] = ('223-043-3406','(541) 754-3010','1-541-754-3010','001-541-754-3010')
```

Comment:
@williamkwon97 This 'tuple' of phone numbers resolves to just one:
```
> a=('223-043-3406','(541) 754-3010','1-541-754-3010','001-541-754-3010')
> a
'001-541-754-3010'
```

Reply:
@rstijerina would this be the result of the way python syntax is being used to populate the field? If we want to test the text used in the expression, we may want to wrap quotes around the text we want to test.

Reply:
@xandroc  what do you mean wrap quotes around the text we want to test? (in this case, we want to test multiple numbers in same test)

Reply:
If you're trying to test a user entering multiple phone numbers in the field (each separated by a comma), maybe something like, ```stateWithFields['data']['demographics']['phone'] = '223-043-3406,(541) 754-3010,1-541-754-3010,001-541-754-3010'```

Reply:
If you want to test different patterns in the same test, you'll have to set each pattern you want to test individually, then rerender the output before testing.

Reply:
@xandroc  stateWithFields['data']['demographics']['phone'] = '223-043-3406,(541) 754-3010,1-541-754-3010,001-541-754-3010' this would give me test result for only first number. I add codes to change number and re rendered but seem like not effective, but I see it changed when I run` console.log(stateWithFields['data']['demographics']['phone'] = '321')`

Reply:
@williamkwon97 you may be able to leverage `test.each` here and pass parameters to a function that is called iteratively for each param you want to test. With this type of solution you should be able to pass in both params and expected result while minimizing code duplication. 

I'd start by getting a single test to pass/fail as expected and then wrap the code similar to the example [here](https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m), and the [docs](https://jestjs.io/docs/api#testeachtablename-fn-timeout)